keep/action/action.py

```diff
@@ -22,8 +24,13 @@ def __init__(
         self.action_config = config
         self.io_handler = IOHandler()
         self.context_manager = ContextManager.get_instance()
+        self.state_manager = StateManager.get_instance()
 
     def run(self):
+        throttled = self._check_throttling(self.action_config.get("name"))
+        if throttled:
+            self.logger.debug("Action %s is throttled", self.action_config.get("name"))
+            return
```

Comment:
this is something i'd consider moving to the alertmanager/alert rather than inside the action itself but idk (could a step be throttled as well?)

* not really important

Reply:
let's keep it that way for now, I would say that if step is throttle we will use another mechanism.